lambdas/service/app.py

```diff
@@ -1301,8 +1301,32 @@ def get_summary():
                                              authentication=request.authentication)
 
 
-def manifest_route(*, fetch: bool, method: str):
-    initiate = method in {'PUT', 'POST'}
+def manifest_route(*, fetch: bool, initiate: bool, curl: bool = False):
+    """
+    Return a decorator of a view handler function in a Chalice application used
+    to generate a file manifest.
+
+    :param fetch: True for an endpoint that returns the status as part of a JSON
+                  response body, False for one that returns the status as a HTTP
+                  response status code.
+
+    :param initiate: True for an endpoint to start a manifest request, False
+                     for one to continue a manifest request.
+
+    :param curl: True for an POST endpoint compatible with curl's ability to
+                 automatically switch to GET when following a 3xx response from
+                 a POST request. To be True requires that `fetch` is False and
+                 `initiate` is True.
+    """
```

Comment:
I appreciate the intent but there are a some of inaccuracies in this docstring that we don't have the time to address at this point. Please remove.